UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -553,12 +563,14 @@ private void FilterPrimitives(Transform transform, out GameObject[] primitives,
 			var nonPrims = new List<GameObject>(childCount);
 
 			// add another primitive if the root object also has a mesh
-			if (transform.gameObject.GetComponent<MeshFilter>() != null
-				&& transform.gameObject.GetComponent<MeshRenderer>() != null)
+			if (transform.gameObject.activeSelf)
```

Comment:
Why is this check needed?

Reply:
This ensures only what is active is exported. I found this to be useful from a user perspective.

Reply:
This is a breaking change since it's different from the current behavior.  But I agree that it seems like it could be useful.  I'll take your change but then make another on top of it to wrap this behavior in an option object that the caller can pass in at runtime.